### PR TITLE
Fix javaweb examples

### PR DESCRIPTION
**Pull request by ldx**

### CODE REVIEW

File: staging/javaweb-tomcat-sidecar/javaweb.yaml [hunk 1]
```diff
@@ -6,6 +6,7 @@ spec:
   initContainers:
     - image: resouer/sample:v1
       name: war
+      command: ["sh", "-c", "cp /sample.war /app"]
```
Comment (ahmetb) [inline staging/javaweb-tomcat-sidecar/javaweb.yaml]: can't this just be `["cp", "/sample.war", "/app"]`?

File: staging/javaweb-tomcat-sidecar/javaweb-2.yaml [hunk 2]
```diff
@@ -6,10 +6,10 @@ spec:
   initContainers:
   - image: resouer/sample:v2
     name: war
-      command:
-        - "cp"
-        - "/sample.war"
-        - "/app"
+    command:
+      - "cp"
```
Comment (ahmetb) [inline staging/javaweb-tomcat-sidecar/javaweb-2.yaml]: let's make it consistent like:

    command: ["cp", "/sample.war", "/app"]

**Comment by ldx** [comment]
@ahmetb thanks for the suggestion, pushed a commit with the updated `command:` line.

**Pull request merged**